apidoc/Facebook.yml

```diff
@@ -227,134 +380,249 @@ description: |
 
-        This is a fallback for when [presentShareDialog](Modules.Facebook.presentShareDialog) is not
-        available.
+        This is a fallback for when Share dialog is not available.
 
-        A `shareCompleted` event is generated to indicate successful or unsuccessful attempt.
+        Listen for the <Modules.Facebook.shareCompleted> to be notified if the attempt was
+        successful or not.
     parameters:
       - name: params
         summary: |
-            A dictionary object containing parameters required by the call, if any.
-            For example
-            fb.presentWebShareDialog({
-                link: 'https://appcelerator.com/',
-                name: 'great product',
-                description: 'Titanium is a great product',
-                caption: 'it rocks too',
-                picture: 'http://www.appcelerator.com/wp-content/uploads/scale_triangle1.png'
-            });
-        type: Object
+            A dictionary object containing optional parameters.
+        type: ShareDialogParams
+    since: 4.0.0
 
   - name: presentSendRequestDialog
     summary: |
-        Opens a supported Facebook dialog.
+        Opens an App Request dialog.
+    description: |
+        A `requestDialogCompleted` event is generated to indicate if the request attempt was successful or unsuccessful.
+    parameters:
+      - name: params
+        summary: |
+            A dictionary object containing following properties:
+
+              * `message` (String): Required. Message to send with the request.
+              * `data` (String): Optional. Additional data to send with the request object.
+        type: Dictionary
+    since: 4.0.0
+
+  - name: reauthorize
+    summary: Makes a request to Facebook for additional permissions.
+    deprecated:
+        since: 4.0.0
+        removed: 4.0.0
+    description: |
+        iOS 6's facebook login forbids minimum authorization to include write permissions.
+        In order to make any write actions, the app must reauthorize asking for the additional
+        permissions. If the application already has these permissions, there will be no user
+        interaction.
+    platforms: [iphone, ipad]
+    parameters:
+      - name: permissions
+        summary: |
+            Array of additional permissions to request. For a complete list of permissions, see the
+            [official Facebook Permissions Reference](http://developers.facebook.com/docs/reference/api/permissions/)
+        type: Array<String>
 
-        For a list of dialogs, parameters, and response formats, see the 
-        [official documentation for Facebook Dialogs](https://developers.facebook.com/docs/android/send-requests).  
+      - name: audience
+        summary: |
+            The extent of the visibility write permissions will have. The value of audience should be one of "me", "friends", or "everyone"
```

Comment:
I think this should be the properties listed below. eg AUDIENCE_ONLY_ME. This is not type String.